x/clp/client/cli/tx.go

```diff
@@ -0,0 +1,43 @@
+package clp
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/thorchain/THORChain/x/clp"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/wire"
+	authcmd "github.com/cosmos/cosmos-sdk/x/auth/client/cli"
+)
+
+// run the test transaction
+func TestTxCmd(cdc *wire.Codec) *cobra.Command {
```

Comment:
Might be a bit misleading to start a func with `Test`, which signals to go that it is a test.

Reply:
But no need to change it, will be removed later anyways, right?